PWAWidgets/README.md

```diff
@@ -212,6 +212,7 @@ One or more Widgets are defined within the `widgets` member of a Web App Manifes
   "data": "/widgets/data/agenda.ical",
   "type": "text/calendar",
   "auth": true,
+  "prevent_multiple": false,
```

Comment:
What a bout "singleton" or "single_instance" instead? Would be affirmation rather than negation.

Reply:
do you think multiple instances of a widget will be the default for many Apps? We were also discussing of using `allow_multiple` and set it to `False` by default.

Reply:
Hard to tell to be honest. Maybe `allow_multiple` or just `multiple`?

Reply:
Thanks! I'll change it to `multiple`.